### PR TITLE
4718 sort papers by year

### PR DESCRIPTION
**Pull request by felixgondwe**

### CODE REVIEW

File: .cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ragingsquirrel3) [inline .cache/v/cache/lastfailed]: Please delete this file.

File: production_asset_url.json [hunk 2]
```diff
@@ -1 +1 @@
-{"url":"https://d1x6jdqbvd5dr.cloudfront.net/e56291feb02a9a5a7a3c"}
\ No newline at end of file
+{"url":"https://d1x6jdqbvd5dr.cloudfront.net/ddd3db6372e540abba68"}
```
Comment (ragingsquirrel3) [inline production_asset_url.json]: Need to rebase from development and deploy assets again. Sorry!

File: .cache/v/cache/lastfailed (hunk 1 above)
Comment (felixgondwe) [reply .cache/v/cache/lastfailed]: will try to add it to .gitignore. It's some vscode editor thing

File: production_asset_url.json (hunk 2 above)
Comment (felixgondwe) [reply production_asset_url.json]: no worries, will do that. Thanks.

**Pull request merged**